dhall.cabal

```diff
@@ -144,6 +144,8 @@ Test-Suite test
         Util
     Build-Depends:
         base               >= 4        && < 5   ,
+        containers         >= 0.5.7.1  && < 0.6 ,
```

Comment:
You can lower the lower bound on `containers` to `>= 0.5.0.0` (consistent with the `containers` dependency for the main library) since that version still has `Data.Map.fromList`